components/integrations/abstract_outgoing_webhook.tsx

```diff
@@ -168,68 +142,75 @@ export default class AbstractOutgoingWebhook extends React.PureComponent {
         const hook = {
             team_id: this.props.team.id,
             channel_id: this.state.channelId,
-            trigger_words: triggerWords,
-            trigger_when: parseInt(this.state.triggerWhen, 10),
-            callback_urls: callbackUrls,
+            trigger_words: this.state.triggerWords.split('\n'),
+            trigger_when: (this.state.triggerWhen, 10),
+            callback_urls: this.state.callbackUrls.split('\n'),
             display_name: this.state.displayName,
             content_type: this.state.contentType,
             description: this.state.description,
             username: this.state.username,
             icon_url: this.state.iconURL,
+            id: '',
+            token: '',
+            create_at: 0,
+            update_at: 0,
+            delete_at: 0,
+            creator_id: '',
+
```

Comment:
how did we derive these properties?

Reply:
These values are not being used in this particular file. But if we do not mention them we get an error because these are necessary properties mentioned in integration.ts so for all the string properties an entry string was included similarly for numeric properties 0 was added